src/load/load.py

```diff
@@ -81,6 +82,41 @@ class VaultArgs:
     kvv2_mount_point: str
 
 
+def load_secrets(vault_args, project, sa_token_path, secrets_file):
+    """load secrets from  Vault and write them to a file"""
+    logging.debug(
+        f"Loading secrets from {vault_args.host}:{vault_args.port}"
+        f"under {vault_args.kvv2_mount_point}"
```

Comment:
nice. didn't know lines can be broken down this way. i'll add a space after the first line, as:
```
DEBUG:root:Loading secrets from vault.stg.artsy.systems:8300under kvv2
```